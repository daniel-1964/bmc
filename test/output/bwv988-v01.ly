% Automatically generated by BMC, the braille music compiler
\version "2.14.2"
\include "articulate.ly"
music =
  <<
    \new PianoStaff <<
      \new Staff = "RH" {
        \clef "treble"
        \key g \major
        \time 3/4
        g'16-4 fis' g'8~ g'16[ d' e' fis'] g'-1[ a' b' cis''] | % 1
        d''16 cis''-3 d''8~ d''16[ a' b' cis''] d''-1[ e'' fis'' d''] | % 2
        g''16 fis'' g''8~ g''16[ fis'' e'' d''] cis''-3[ e'' a'-2 g'] | % 3
        fis'16-3[ e' d' cis'-2] d'[ fis' a g-3] fis a d8 | % 4
        r8 d''16-5 c'' d''8 g'-2 b d'' | % 5
        r8 e''16-5 d'' e''8 a' c' e'' | % 6
        r8 fis''16-4 e'' fis''8 d'' a'' c''~-1 | % 7
        c''8 b' r16[ g' b' d''] g''[ d''-1 g''-3 a''] | % 8
        b''16[ g''-4 d'' b'] g'-2[ b'-1 d'' g''] b''[ g''-3 fis'' e''] | % 9
        a''16-5[ e'' cis'' a'] fis'[ a'-1 cis'' e''] a''[ fis'' e'' d''] | % 10
        g''16[ d'' b' g'] e'[ g' b' d''] g''[ fis''-3 e'' d''] | % 11
        cis''16-4[ g' e' cis'-4] a[ cis' e'-1 g'] cis''[ e'' d'' cis''] | % 12
        d''8 fis-2 fis a'-1 d''-2 fis'' | % 13
        b'8 g g b'-1 e''-3 g'' | % 14
        cis''16-2[ e'' a' g'-3] fis'[ a'-1 d'' fis''] g''[ e'' d''-1 cis''-2] | % 15
        fis''16-5[ d''-1 cis''-4 b'] a'[ g' fis' e'] d'4  \bar ":|:" % 16
         fis''16-2 g'' a''8~ a''16[ b'' a'' g''] fis''[ e'' d''-4 c''] | % 17
        b'16 c'' d''8~ d''16[ e'' d'' c''] b'[ a' g'-3 fis'] | % 18
        e'16[ gis' a' b'] a'[ e' a'-2 b'] c''[ a'-1 dis''-3 e''] | % 19
        fis''16[ e'' dis'' cis''] b'2~ | % 20
        b'16 dis''-4 e''8~ e''16 dis'-2 e'8~ e'16 dis-2 e8 | % 21
        r16 gis''-4 a''8~ a''16 gis'-2 a'8~ a'16 gis-2 a8~-1 | % 22
        a16[ b c' fis'] b[ dis' e' g'] fis'-4[ e' dis' a'] | % 23
        g'16[ fis' e'-1 dis'-2] e'-3[ g' b a-3] g b e8 | % 24
        r8 e''-3 c''-2 e'' a'' a' | % 25
        r8 d'' b' d'' g'' g' | % 26
        c''16-5[ a' e' c'] a-2[ c'-1 e' a'] c''[ a'-1 c''-2 e''] | % 27
        fis''16[ c'' a' fis'-3] d'-2[ fis' a'-1 c''] fis''[ c''-1 fis''-2 a''] | % 28
        b''16[ g''-4 d'' b'] g'-2[ b'-1 d'' g''] b''[ f''-1 b''-3 d'''] | % 29
        e''16[ d''' c''' e''-1] d''-2[ c''' b'' d''] c''-2[ e''-1 fis''-3 g''] | % 30
        a''16[ c'' b'-2 a'] b'-3[ d'' b' g'] c''[ a' g' fis'-2] | % 31
        b'16[ g'-1 fis'-4 e'] d'[ c' b a] g4  \bar ":|" % 32
      }
      \new Staff = "LH" {
        \clef "bass"
        \key g \major
        \time 3/4
        g,8 b16-2 a b8 g-1 g, g | % 1
        fis,8 fis16-2 e fis8 d-1 fis,-4 d | % 2
        e,8 e16-1 d-3 e8-2 g a, cis'-1 | % 3
        d8 fis16-2 e fis8 d-1 d, r16 c-2 | % 4
        b,16 a, b,8~ b,16[ d e-4 fis] g[ a b-2 g-1] | % 5
        c16-4 b, c8~ c16[ e-1 fis-4 g] a[ b c'-2 a-1] | % 6
        d16-4 cis d8~ d16[ a b c'] d'[ e' fis' d'] | % 7
        g'16[ fis' g' d'] b[ d'-1 g-3 b] d[ g-1 b,-4 d] | % 8
        g,8 g b-2 g-1 g, g | % 9
        fis,8 fis a fis fis, fis | % 10
        e,8 e g e e, g | % 11
        a,8 e g e a, g | % 12
        fis16-4[ a d' fis'-3] a'-2[ fis' d'-1 a] fis[ a d fis-4] | % 13
        g16-5[ b d' g'] b'-2[ g'-1 d' b] g[ b-1 e g-4] | % 14
        a8 cis'-2 d'16 a fis d a8 a,-2 | % 15
        d,16[ d e-4 fis] g[ a b cis'] d'4  \bar ":|:" % 16
         d,8 fis16-2 e fis8 d-1 d, fis-1 | % 17
        g,8 b16 a b8 g g, b | % 18
        c8 c'16-1 b c'8 fis-4 a c' | % 19
        a8 fis-3 dis16-4[ b, dis-3 fis] b[ dis'-3 fis' a'] | % 20
        g'8. fis'16 g'8. fis16-2 g8. b,16-5 | % 21
        c8. b'16-2 c''8. b16-2 c'8. e16-4 | % 22
        dis8 a-2 g-1 ais,-4 b,-5 fis-2 | % 23
        e8 g16 fis-3 g8 e-1 e, r16 d-3 | % 24
        c16[ e a c'-3] e'-2[ c' a-1 e-2] c[ e d c] | % 25
        b,16[ d g b] d'[ b g d] b,[ d-2 c b,] | % 26
        a,8 c e g fis-3 e | % 27
        d8 fis a c' b-3 a | % 28
        g8 b d' f' e' d' | % 29
        c'8 e'-4 fis' gis' a' g' | % 30
        fis'8 d' g'-1 g d'-1 d | % 31
        g16[ g, a, b,] c[ d e fis] g4  \bar ":|" % 32
      }
    >>
  >>

\score {
  \music
  \layout { }
}
\score {
  \unfoldRepeats \articulate \music
  \midi { }
}
