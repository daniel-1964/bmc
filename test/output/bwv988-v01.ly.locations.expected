% Automatically generated by BMC, the braille music compiler
\version "2.14.2"
\include "articulate.ly"
music =
  <<
    \new PianoStaff <<
      \new Staff = "RH" {
        \clef "treble"
        \key g \major
        \time 3/4
        g'16-4%{5%} fis'%{8%} g'8~%{11%} g'16[%{14%} d'%{17%} e'%{20%} fis']%{23%} g'-1[%{26%} a'%{29%} b'%{32%} cis'']%{35%} | % 1
        d''16%{42%} cis''-3%{45%} d''8~%{48%} d''16[%{51%} a'%{54%} b'%{57%} cis'']%{60%} d''-1[%{63%} e''%{66%} fis''%{69%} d'']%{72%} | % 2
        g''16%{79%} fis''%{82%} g''8~%{85%} g''16[%{88%} fis''%{91%} e''%{94%} d'']%{97%} cis''-3[%{100%} e''%{103%} a'-2%{106%} g']%{109%} | % 3
        fis'16-3[%{116%} e'%{119%} d'%{122%} cis'-2]%{125%} d'[%{128%} fis'%{131%} a%{134%} g-3]%{137%} fis%{140%} a%{143%} d8%{146%} | % 4
        r8%{256%} d''16-5%{259%} c''%{262%} d''8%{265%} g'-2%{268%} b%{271%} d''%{274%} | % 5
        r8%{279%} e''16-5%{282%} d''%{285%} e''8%{288%} a'%{291%} c'%{294%} e''%{297%} | % 6
        r8%{302%} fis''16-4%{305%} e''%{308%} fis''8%{311%} d''%{314%} a''%{317%} c''~-1%{320%} | % 7
        c''8%{327%} b'%{330%} r16[%{331%} g'%{334%} b'%{337%} d'']%{340%} g''[%{343%} d''-1%{346%} g''-3%{349%} a'']%{352%} | % 8
        b''16[%{514%} g''-4%{517%} d''%{520%} b']%{523%} g'-2[%{526%} b'-1%{529%} d''%{532%} g'']%{535%} b''[%{538%} g''-3%{541%} fis''%{544%} e'']%{547%} | % 9
        a''16-5[%{554%} e''%{557%} cis''%{560%} a']%{563%} fis'[%{566%} a'-1%{569%} cis''%{572%} e'']%{575%} a''[%{578%} fis''%{581%} e''%{584%} d'']%{587%} | % 10
        g''16[%{594%} d''%{597%} b'%{600%} g']%{603%} e'[%{606%} g'%{609%} b'%{612%} d'']%{615%} g''[%{618%} fis''-3%{621%} e''%{624%} d'']%{627%} | % 11
        cis''16-4[%{634%} g'%{637%} e'%{640%} cis'-4]%{643%} a[%{646%} cis'%{649%} e'-1%{652%} g']%{655%} cis''[%{658%} e''%{661%} d''%{664%} cis'']%{667%} | % 12
        d''8%{766%} fis-2%{769%} fis%{772%} a'-1%{775%} d''-2%{778%} fis''%{781%} | % 13
        b'8%{788%} g%{791%} g%{794%} b'-1%{797%} e''-3%{800%} g''%{803%} | % 14
        cis''16-2[%{810%} e''%{813%} a'%{816%} g'-3]%{819%} fis'[%{822%} a'-1%{825%} d''%{828%} fis'']%{831%} g''[%{834%} e''%{837%} d''-1%{840%} cis''-2]%{843%} | % 15
        fis''16-5[%{850%} d''-1%{853%} cis''-4%{856%} b']%{859%} a'[%{862%} g'%{865%} fis'%{868%} e']%{871%} d'4%{874%}  \bar ":|:" % 16
         fis''16-2%{1024%} g''%{1027%} a''8~%{1030%} a''16[%{1033%} b''%{1036%} a''%{1039%} g'']%{1042%} fis''[%{1045%} e''%{1048%} d''-4%{1051%} c'']%{1054%} | % 17
        b'16%{1061%} c''%{1064%} d''8~%{1067%} d''16[%{1070%} e''%{1073%} d''%{1076%} c'']%{1079%} b'[%{1082%} a'%{1085%} g'-3%{1088%} fis']%{1091%} | % 18
        e'16[%{1098%} gis'%{1101%} a'%{1104%} b']%{1107%} a'[%{1110%} e'%{1113%} a'-2%{1116%} b']%{1119%} c''[%{1122%} a'-1%{1125%} dis''-3%{1128%} e'']%{1131%} | % 19
        fis''16[%{1138%} e''%{1141%} dis''%{1144%} cis'']%{1147%} b'2~%{1150%} | % 20
        b'16%{1157%} dis''-4%{1160%} e''8~%{1163%} e''16%{1166%} dis'-2%{1169%} e'8~%{1172%} e'16%{1175%} dis-2%{1178%} e8%{1181%} | % 21
        r16%{1321%} gis''-4%{1324%} a''8~%{1327%} a''16%{1330%} gis'-2%{1333%} a'8~%{1336%} a'16%{1339%} gis-2%{1342%} a8~-1%{1345%} | % 22
        a16[%{1352%} b%{1355%} c'%{1358%} fis']%{1361%} b[%{1364%} dis'%{1367%} e'%{1370%} g']%{1373%} fis'-4[%{1376%} e'%{1379%} dis'%{1382%} a']%{1385%} | % 23
        g'16[%{1392%} fis'%{1395%} e'-1%{1398%} dis'-2]%{1401%} e'-3[%{1404%} g'%{1407%} b%{1410%} a-3]%{1413%} g%{1416%} b%{1419%} e8%{1422%} | % 24
        r8%{1427%} e''-3%{1430%} c''-2%{1433%} e''%{1436%} a''%{1439%} a'%{1442%} | % 25
        r8%{1447%} d''%{1450%} b'%{1453%} d''%{1456%} g''%{1459%} g'%{1462%} | % 26
        c''16-5[%{1857%} a'%{1860%} e'%{1863%} c']%{1866%} a-2[%{1869%} c'-1%{1872%} e'%{1875%} a']%{1878%} c''[%{1881%} a'-1%{1884%} c''-2%{1887%} e'']%{1890%} | % 27
        fis''16[%{1897%} c''%{1900%} a'%{1903%} fis'-3]%{1906%} d'-2[%{1909%} fis'%{1912%} a'-1%{1915%} c'']%{1918%} fis''[%{1921%} c''-1%{1924%} fis''-2%{1927%} a'']%{1930%} | % 28
        b''16[%{1937%} g''-4%{1940%} d''%{1943%} b']%{1946%} g'-2[%{1949%} b'-1%{1952%} d''%{1955%} g'']%{1958%} b''[%{1961%} f''-1%{1964%} b''-3%{1967%} d''']%{1970%} | % 29
        e''16[%{1977%} d'''%{1980%} c'''%{1983%} e''-1]%{1986%} d''-2[%{1989%} c'''%{1992%} b''%{1995%} d'']%{1998%} c''-2[%{2001%} e''-1%{2004%} fis''-3%{2007%} g'']%{2010%} | % 30
        a''16[%{2017%} c''%{2020%} b'-2%{2023%} a']%{2026%} b'-3[%{2029%} d''%{2032%} b'%{2035%} g']%{2038%} c''[%{2041%} a'%{2044%} g'%{2047%} fis'-2]%{2050%} | % 31
        b'16[%{2057%} g'-1%{2060%} fis'-4%{2063%} e']%{2066%} d'[%{2069%} c'%{2072%} b%{2075%} a]%{2078%} g4%{2081%}  \bar ":|" % 32
      }
      \new Staff = "LH" {
        \clef "bass"
        \key g \major
        \time 3/4
        g,8%{153%} b16-2%{156%} a%{159%} b8%{162%} g-1%{165%} g,%{168%} g%{171%} | % 1
        fis,8%{178%} fis16-2%{181%} e%{184%} fis8%{187%} d-1%{190%} fis,-4%{193%} d%{196%} | % 2
        e,8%{203%} e16-1%{206%} d-3%{209%} e8-2%{212%} g%{215%} a,%{218%} cis'-1%{221%} | % 3
        d8%{228%} fis16-2%{231%} e%{234%} fis8%{237%} d-1%{240%} d,%{243%} r16%{244%} c-2%{247%} | % 4
        b,16%{359%} a,%{362%} b,8~%{365%} b,16[%{368%} d%{371%} e-4%{374%} fis]%{377%} g[%{380%} a%{383%} b-2%{386%} g-1]%{389%} | % 5
        c16-4%{396%} b,%{399%} c8~%{402%} c16[%{405%} e-1%{408%} fis-4%{411%} g]%{414%} a[%{417%} b%{420%} c'-2%{423%} a-1]%{426%} | % 6
        d16-4%{433%} cis%{436%} d8~%{439%} d16[%{442%} a%{445%} b%{448%} c']%{451%} d'[%{454%} e'%{457%} fis'%{460%} d']%{463%} | % 7
        g'16[%{470%} fis'%{473%} g'%{476%} d']%{479%} b[%{482%} d'-1%{485%} g-3%{488%} b]%{491%} d[%{494%} g-1%{497%} b,-4%{500%} d]%{503%} | % 8
        g,8%{674%} g%{677%} b-2%{680%} g-1%{683%} g,%{686%} g%{689%} | % 9
        fis,8%{696%} fis%{699%} a%{702%} fis%{705%} fis,%{708%} fis%{711%} | % 10
        e,8%{718%} e%{721%} g%{724%} e%{727%} e,%{730%} g%{733%} | % 11
        a,8%{740%} e%{743%} g%{746%} e%{749%} a,%{752%} g%{755%} | % 12
        fis16-4[%{881%} a%{884%} d'%{887%} fis'-3]%{890%} a'-2[%{893%} fis'%{896%} d'-1%{899%} a]%{902%} fis[%{905%} a%{908%} d%{911%} fis-4]%{914%} | % 13
        g16-5[%{921%} b%{924%} d'%{927%} g']%{930%} b'-2[%{933%} g'-1%{936%} d'%{939%} b]%{942%} g[%{945%} b-1%{948%} e%{951%} g-4]%{954%} | % 14
        a8%{961%} cis'-2%{964%} d'16%{967%} a%{970%} fis%{973%} d%{976%} a8%{979%} a,-2%{982%} | % 15
        d,16[%{989%} d%{992%} e-4%{995%} fis]%{998%} g[%{1001%} a%{1004%} b%{1007%} cis']%{1010%} d'4%{1013%}  \bar ":|:" % 16
         d,8%{1188%} fis16-2%{1191%} e%{1194%} fis8%{1197%} d-1%{1200%} d,%{1203%} fis-1%{1206%} | % 17
        g,8%{1213%} b16%{1216%} a%{1219%} b8%{1222%} g%{1225%} g,%{1228%} b%{1231%} | % 18
        c8%{1238%} c'16-1%{1241%} b%{1244%} c'8%{1247%} fis-4%{1250%} a%{1253%} c'%{1256%} | % 19
        a8%{1263%} fis-3%{1266%} dis16-4[%{1269%} b,%{1272%} dis-3%{1275%} fis]%{1278%} b[%{1281%} dis'-3%{1284%} fis'%{1287%} a']%{1290%} | % 20
        g'8.%{1297%} fis'16%{1300%} g'8.%{1303%} fis16-2%{1306%} g8.%{1309%} b,16-5%{1312%} | % 21
        c8.%{1469%} b'16-2%{1472%} c''8.%{1475%} b16-2%{1478%} c'8.%{1481%} e16-4%{1484%} | % 22
        dis8%{1491%} a-2%{1494%} g-1%{1497%} ais,-4%{1500%} b,-5%{1503%} fis-2%{1506%} | % 23
        e8%{1513%} g16%{1516%} fis-3%{1519%} g8%{1522%} e-1%{1525%} e,%{1528%} r16%{1529%} d-3%{1532%} | % 24
        c16[%{1539%} e%{1542%} a%{1545%} c'-3]%{1548%} e'-2[%{1551%} c'%{1554%} a-1%{1557%} e-2]%{1560%} c[%{1563%} e%{1566%} d%{1569%} c]%{1572%} | % 25
        b,16[%{1579%} d%{1582%} g%{1585%} b]%{1588%} d'[%{1591%} b%{1594%} g%{1597%} d]%{1600%} b,[%{1603%} d-2%{1606%} c%{1609%} b,]%{1612%} | % 26
        a,8%{2088%} c%{2091%} e%{2094%} g%{2097%} fis-3%{2100%} e%{2103%} | % 27
        d8%{2110%} fis%{2113%} a%{2116%} c'%{2119%} b-3%{2122%} a%{2125%} | % 28
        g8%{2132%} b%{2135%} d'%{2138%} f'%{2141%} e'%{2144%} d'%{2147%} | % 29
        c'8%{2154%} e'-4%{2157%} fis'%{2160%} gis'%{2163%} a'%{2166%} g'%{2169%} | % 30
        fis'8%{2176%} d'%{2179%} g'-1%{2182%} g%{2185%} d'-1%{2188%} d%{2191%} | % 31
        g16[%{2198%} g,%{2201%} a,%{2204%} b,]%{2207%} c[%{2210%} d%{2213%} e%{2216%} fis]%{2219%} g4%{2222%}  \bar ":|" % 32
      }
    >>
  >>

\score {
  \music
  \layout { }
}
\score {
  \unfoldRepeats \articulate \music
  \midi { }
}
